aspnetcore/security/cors.md

```diff
@@ -487,7 +487,7 @@ To test CORS:
 
      **Access to XMLHttpRequest at `https://webapi.azurewebsites.net/api/values/1` from origin `https://localhost:44375` has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.**
      
-CORS-enabled endpoints can be tested with a tool, such as [Fiddler](https://www.telerik.com/fiddler) or [Postman](https://www.getpostman.com/). When using a tool, the origin of the request specififed by the `Origin` header must differ from the host receiving the request. When the `Origin` header isn't present or indicates the same host, the CORS Middleware assumes that the request is from the same origin. Since the request isn't *cross-origin*:
+CORS-enabled endpoints can be tested with a tool, such as [Fiddler](https://www.telerik.com/fiddler) or [Postman](https://www.getpostman.com/). When using a tool, the origin of the request specififed by the `Origin` header must differ from the host receiving the request. If the request isn't *cross-origin* based on the value of the the `Origin` header:
```

Comment:
Duplicate "the the"

Reply:
ooooops ..... this happens more than you'd think. When looking at these in an editor (most of us use VS Code) and the line breaks right in the middle of the double-word, it's *hard to spot*. Thanks for catching :eyes: that!

Reply:
```suggestion
CORS-enabled endpoints can be tested with a tool, such as [Fiddler](https://www.telerik.com/fiddler) or [Postman](https://www.getpostman.com/). When using a tool, the origin of the request specififed by the `Origin` header must differ from the host receiving the request. If the request isn't *cross-origin* based on the value of the `Origin` header:
```

Reply:
Jerks!!! lol ..... they won't let me commit my own suggestion. 😠